style.css

```
@@ -0,0 +1,161 @@
+/*font-family: 'Big Shoulders Display', cursive;
+font-family: 'Lexend Deca', sans-serif;*/
+
+body {
+    background-color: hsl(0, 0%, 95%);
+    padding: 0;
+}
+
+.container {
+    width: 70%;
+    height: 80vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 50px auto;
+}
+
+.cards {
+    width: 100%;
+    display: grid;
+    overflow: hidden;
+    grid-template-columns: repeat(3, 1fr);
+    text-align: left;
+}
+
+.card-one {
```

```suggestion
.card {
    padding: 70px 50px; 
}
.card-one {
    background-color: hsl(31, 77%, 52%);
}
.card-two {
    background-color: hsl(184, 100%, 22%);
}

.card-three {
    padding: 70px 50px;
    background-color: hsl(179, 100%, 13%);
}
```


One way to fix the responsive issues is to play around with the padding sizes at that screen size. 

1. Since card one, two and three all were sharing the same padding styles, I have removed them and put them in one class that can be used to target all the three cards. 

2. This way it keeps you from writing duplicate code and makes it easier to make the responsive changes. e.g if you have the padding as `padding: 70px 30px;` the buttons render well.

---

Ooowk

---

I will commit the changes now

---

@gertie-sheshe I think we're good here, smiles